packages/babel-helpers/src/helpers.ts

```diff
@@ -289,9 +289,10 @@ helpers.defineProperty = helper("7.0.0-beta.0")`
   }
 `;
 
+// need a bind because https://github.com/babel/babel/issues/14527
 helpers.extends = helper("7.0.0-beta.0")`
   export default function _extends() {
-    _extends = Object.assign || function (target) {
+    _extends = Object.assign.bind() || function (target) {
       for (var i = 1; i < arguments.length; i++) {
         var source = arguments[i];
```

Comment:
Oops, I suddenly found that an exception is thrown when `Object.assign==undefined`.

Reply:
Good catch! We should use `? ... :` like we do for `Object.getPrototypeOf`.

Reply:
Sorry, I forgot that!